mars/oscar/backends/ray/utils.py

```diff
@@ -28,26 +90,23 @@ def address_to_placement_group_bundle(address):
         while tmp and tmp != "/":
             tmp, item = posixpath.split(tmp)
             parts.append(item)
-    if parts and len(parts) != 1:
-        raise ValueError(f"Only bundle index path is allowed in ray address {address}.")
-    name, bundle_index = parsed_url.netloc, parts[-1] if parts else ""
-    if bool(name) != bool(bundle_index):
-        raise ValueError(f"Missing placement group name or bundle index from address {address}")
-    if name and bundle_index:
-        return name, int(bundle_index)
-    else:
-        return name, -1
+    parts = list(reversed(parts))
+    return parsed_url.netloc, parts
+
+
+def process_placement_to_address(pg_name: str, bundle_index: int, process_index: int = 0):
+    return f"ray://{pg_name}/{bundle_index}/{process_index}"
 
 
-def placement_group_bundle_to_address(pg_name, bundle_index):
+def node_placement_to_address(pg_name, bundle_index):
     return f"ray://{pg_name}/{bundle_index}"
 
 
-def addresses_to_placement_group_info(address_to_resources):
+def node_addresses_to_pg_info(address_to_resources):
```

Comment:
The `node_addresses_to_pg_info` is not clear, and the naming rules are inconsistent.

We can abstract some concepts: `node address`, `process address`, `placement group`, `placement group info`, `placement info`. Then naming the utils by these concepts.
- node_address_to_placement_info
- placement_info_to_node_address
- process_address_to_placement_info
- placement_info_to_process_address
- node_addresses_to_placement_group_info
- placement_group_info_to_node_addresses

It's just an example. The name is longer but clearer.